### ISSUE TITLE
🔧 TV Show jobs tailoring of names and tracks to rip

### ISSUE THREAD

**Issue by flammableliquids** [automatic-ripping-machine/automatic-ripping-machine#805]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

No.

### Describe the solution you'd like?

When Ripping TV shows, it is problematic, the methodology currently employed creates additional work which goes against the goal of the system being as 'automatic' as possible.

As such I believe the following features would reduce the requirement to go in and finesse (Rename and delete unwanted files) a job at command line:

1.  
 - 
 - 
2. If a job is identified as a TV_Show, allow a page that has the following abiliites 

- [ ] Allow a job to be identified as TV_Show  
- [ ] TV_Show: In the Title Search Menu - allow above check box. If ticked: 
turn off 'main feature' option for the job. 
visually allow for the fields 'Disk Number' and 'Season number' and then concat those two fields into the Job name.
- [ ] TV_Show: In the Custom Title Search Menu (See Above)
- [ ] Per job: allow the GUI to specify which tracks are to be ripped (Regardless of TV or Movie). 
Rip as per standard style.
- [ ]  Per job: allow the GUI to specify the names of tracks. 
(This feature is useful, so that when you insert a disk, you can look at the box, enter the names of the special features, OR the names of individual episodes of a show)

### Describe alternatives you've considered?

Command line file rename is the only option

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by microtechno9000** [comment]
Agree, we need to do something with how TV Series are managed. I have ripped a few seasons and get complaints from the house because they don't match the Jellyfin data.

I have started working on a "session" style automation for jobs, to allow for the different media types. However I havent had much chance lately to work on it. Once I make some progress will add a link to this issue.

**Comment by microtechno9000** [comment]
This is something that I would like to implement, and seems there are a few users who would also be keen. If you have a look at the community discussion #815 I have a proposed solution.
Would be keen to hear your feedback

**Comment by microtechno9000** [comment]
@flammableliquids I have rolled this into the ARM Sessions project for incorporation.
